### ISSUE TITLE
VS Code does not open files

### ISSUE THREAD

**Issue by videche92** [microsoft/vscode#180167]
Type: <b>Bug</b>

Whenever I try to open a File (.log, .py, .txt, etc) from VS CODE, nothing happens. I can go to File -> Open File -> Select the File but nothing really happens... Also if I double click on a File that is set to open with VS CODE, the program will open but the file will not show up in VS CODE.

VS Code version: Code 1.77.3 (704ed70d4fd1c6bd6342c436f1ede30d1cff4710, 2023-04-12T09:16:02.548Z)
OS version: Windows_NT x64 10.0.22000
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|63.74GB (45.83GB free)|
|Process Argv|C:\\Users\\mavidech\\Downloads\\prcalypsappsvc01-InspectIaaSDisk-ju3ob0sj.bly_7c79beb081\\device_0\\var\\log\\waagent.log --crash-reporter-id 3ecc732a-4a9d-451b-8fe4-10e6763a31f4|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2023.6.0
vscode-pylance|ms-|2023.4.20
jupyter|ms-|2023.3.1201040234
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
markdown-preview-enhanced|shd|0.6.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
a9j8j154:30646983
showlangstatbar:30672706
vsctsb:30705553
functionswalk:30687959
pythonms35:30701012
pythonfmttextcf:30716743
pythonclientmv:30713943

```

</details>

**Comment by videche92** [comment]
Hi, I'm still facing the issue and I really need to get this working ASAP as I use it in my daily basis... Is there any fix for it?

**Comment by deepak1556** [comment]
Can you check if the issue repros with our latest insiders https://code.visualstudio.com/insiders ?

**Comment by videche92** [comment]
Thanks for the response, I just tested it and it works fine with the insiders.

**Comment by deepak1556** [comment]
Thanks for confirming, this should be addressed with our next stable `v1.78` that will get released this week.
